src/components/member-profile/index.js

```diff
@@ -372,6 +373,29 @@ const Profile = (props) => {
               className={classNames.profilePic}
               alt={fullName}
             />
+            {isSuperUser &&
+              (isMember ? (
+                <div className={classNames.memberStatus}>
+                  <img
+                    alt="info icon"
+                    src="icons/info.png"
+                    width={20}
+                    height={20}
+                  />
+                  <span>User is a Member</span>
+                </div>
+              ) : (
+                <div className={classNames.memberStatus}>
+                  <img
+                    alt="info icon"
+                    src="icons/info.png"
+                    width={20}
+                    height={20}
+                  />
+                  <span>User is not a Member</span>
+                </div>
+              ))}
```

Comment:
rather than repeating the same code, let's create a message variable which will set that message based on if the user is a member or not

Reply:
oh I missed that, here's how it looks now 
![image](https://user-images.githubusercontent.com/70599880/200818456-19197bed-e31d-4cd6-b46f-ea1a132c1526.png)